electron/libelectron/src/backends/memory_mgmt.hh

```diff
@@ -0,0 +1,32 @@
+#include <stdlib.h>
+#include <string.h>
+#include <stdio.h>
+#include <assert.h>
+#include <memory>
+#include <sys/mman.h>
+
+namespace memory_mgmt{
+
+class FirstFitAllocator{
+public:
+struct MemBlock
```

Comment:
Let's try to match style with the rest of the code base:
```cpp
class FirstFitAllocator {
public:
    struct MemBlock {
    //...
    };
    
    FirstFitAllocator(size_t sz);
    //...
};
```

Reply:
Done